lib/asciidoctor-pdf/converter.rb

```diff
@@ -336,8 +336,11 @@ def convert_abstract node
     add_dest_for_block node if node.id
     pad_box @theme.abstract_padding do
       theme_font :abstract do
-        # FIXME control first_line_options using theme
-        prose_opts = { line_height: @theme.abstract_line_height, first_line_options: { styles: [font_style, :bold] } }
+        prose_opts = { line_height: @theme.abstract_line_height }
+        # FIXME control more first_line_options using theme
+        if (first_line_font_style = @theme.abstract_first_line_font_style.to_sym) && first_line_font_style != font_style
```

Comment:
Using `.to_sym` in the assignment will throw an exception if this key is not present. Therefore, the `.to_sym` need to be moved to where `first_line_font_style` is referenced once the truthy check passes.